crates/viewer/re_renderer/src/context.rs

```diff
@@ -305,7 +305,7 @@ impl RenderContext {
         if cfg!(target_arch = "wasm32")
             && self.device_caps.backend_type == WgpuBackendType::WgpuCore
         {
-            self.device.poll(wgpu::Maintain::Wait);
+            self.device.poll(wgpu::PollType::Wait);
```

Comment:
Since https://github.com/gfx-rs/wgpu/issues/3601 is closed now this entire block needs to be re-evaluated. Should be possible to have it be removed now, but may require configuring the [fence behavior](https://docs.rs/wgpu/latest/wgpu/struct.GlBackendOptions.html)
Either way this needs to be thoroughly tested on WebGL :)